app-directory/specification/appd.yaml

```diff
@@ -0,0 +1,295 @@
+openapi: 3.0.0
```

Comment:
I am tempted to move this file into a `appd-spec.yaml` file at the root of the appd source, to make it nice and explicit. But leave it as is for now, we need Frank's input.